nano/rpc_test/rpc.cpp

```diff
@@ -1858,16 +1858,148 @@ TEST (rpc, pending)
 		ASSERT_EQ (block4->hash (), hash);
 		ASSERT_EQ (block3->hash (), hash1);
 	}
+}
+
+TEST (rpc, receivable_offset)
```

Comment:
It would be nice to have a comment explaining what this unit test intends to test.